Author: Luca Boccassi <luca.boccassi@gmail.com>
Description: Fix kernel module load on 4.9 and greater
 From kernel 4.9 and newer (commit fa5386459f06) non-modesetting drivers have
 to use the DRM flag DRIVER_LEGACY. Without this flag the kernel module does
 not load correctly.

--- kernel/nv-drm.c
+++ kernel/nv-drm.c
@@ -71,7 +71,11 @@
 };
 
 static struct drm_driver nv_drm_driver = {
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 9, 0)
     .driver_features = 0,
+#else
+    .driver_features = DRIVER_LEGACY,
+#endif
     .load = nv_drm_load,
     .unload = nv_drm_unload,
     .fops = &nv_drm_fops,
